reclaimer/os_hek/defs/actv.py

```diff
@@ -11,13 +11,13 @@
 
 # TODO use some supyr trickery to make this nicer.
 
-# grenades descriptor index in actv_body is 10
+# grenades descriptor index in actv_body is 11
 # grenade_type descriptor index in grenades_desc is 1
 
 # replace the grenade_type descriptor with one
 # that uses open sauce's extra grenade slots
 actv_body = dict(actv_body)
-grenades_desc = actv_body[10] = dict(actv_body[10])
+grenades_desc = actv_body[11] = dict(actv_body[11])
```

Comment:
No, you're not fucking doing this again. You wrote a system for this and now you're still not using it.

This should be a learning moment

Reply:
Don't curse at me over this. I was ready to go to sleep and was not feeling up to writing the code to do this. This was meant to be a band-aid solution until we could get around to this:
https://github.com/Sigmmma/reclaimer/issues/63

I actually wanted to do this when I wasn't so stupid tired.